src/commands/index.ts

```diff
@@ -12,6 +13,9 @@ import { registerPlaygroundCommands } from "./playground";
 import { registerScratchCommands } from "./scratch";
 import { registerTourCommands } from "./tour";
 
+// https://github.com/patriksimek/vm2/issues/70#issuecomment-297601837
+const { NodeVM } = eval("require")("vm2");
```

Comment:
We’d need to add a Webpack abstraction for this, so that this could continue to run in the serverless web.